site-structure.json

```diff
@@ -1,7 +1,18 @@
 [
     {
         "category": "Overview",
-        "categoryHref":"/index.html"
+        "categoryHref":"/index.html",
+        "sections":
+        [
+            {
+                "sectionName":"Preferences Server Integration",
+                "pages":
+                [
+                    { "pageName": "Introduction", "href": "/integrationIntro.html"},
+                    { "pageName": "The Work Flow", "href": "/integrationWorkFlow.html"}
+                ]
+            }
+        ]
     },
```

Comment:
Looking at how this renders, it's not really very clear. How about something like this?
```
        "category": "Overview",
        "categoryHref":"/index.html",
        "sections":
        [
            {
                "sectionName":"First Discovery Tool",
                "pages":
                [
                    { "pageName": "Introduction", "href": "/index.html"}
                ]
            },
            {
                "sectionName":"Preferences Server Integration",
                "pages":
                [
                    { "pageName": "Introduction", "href": "/integrationIntro.html"},
                    { "pageName": "The Work Flow", "href": "/integrationWorkFlow.html"}
                ]
            }
        ]
```